e2e/tests/regression/test_persistentvolumeclaim.robot

```diff
@@ -0,0 +1,44 @@
+*** Settings ***
+Documentation    PersistentVolumeClaim Test Cases
+
+Test Tags    regression
+
```

Comment:
Importing `variables.resource` can help prevent teardown failure when powering nodes on:

https://ci.longhorn.io/job/private/job/longhorn-e2e-test/2067/robot/report/log.html

Reply:
Updated. Thank you!